### PR TITLE
Add advanced tab to settings

### PR DESCRIPTION
**Pull request by MSuih**
Adds a new tab to settings and couple of new options to it, some of them new and some moved from debug tab. Also changes the default value of sleep timer accuracy from host to usleep.

image

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -1039,6 +1042,68 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 	xemu_settings->EnhanceComboBox(ui->netStatusBox, emu_settings::ConnectionStatus);
 	SubscribeTooltip(ui->netStatusBox, json_net["netStatusBox"].toString());
 
+
+	//            _______    _
```
Comment (HerrHulaHoop) [inline rpcs3/rpcs3qt/settings_dialog.cpp]: //                _                               _   _______    _     
	//       /\      | |                             | | |__   __|  | |    
	//      /  \   __| |_   ____ _ _ __   ___ ___  __| |    | | __ _| |__  
	//     / /\ \ / _` \ \ / / _` | '_ \ / __/ _ \/ _` |    | |/ _` | '_ \ 
	//    / ____ \ (_| |\ V / (_| | | | | (_|  __/ (_| |    | | (_| | |_) |
	//   /_/    \_\__,_| \_/ \__,_|_| |_|\___\___|\__,_|    |_|\__,_|_.__/

**Comment by AniLeo** [comment]
Firmware Settings and Firmware Libraries should also be on Advanced IMO

**Comment by elad335** [comment]
Why have you reworded vblank rate on GUI? :thonk:

**Comment by Dokman** [comment]
you will add Forcr CPU-Blit

**Comment by Kravickas** [comment]
who use these buffers options

**Comment by MarioSonic2987** [comment]
@Kravickas Batman: Arkham Origins and Batman: Arkham City need Write Depth Buffers to fix broken graphics.

**Comment by HerrHulaHoop** [comment]
Does `Disable On-Disk Shader Cache` have any real use? Haven't seen anyone use that one ever.

**Comment by Asinin3** [comment]
@HerrHulaHoop Yes, faster start-up time especially for low-end systems. Or in games with a huge number of shaders but you just want to test the title-screen or one small area. Seems more like a question for Discord though.

**Comment by MSuih** [comment]
>Does Disable On-Disk Shader Cache have any real use? Haven't seen anyone use that one ever.

I don't know, but it's not really a debug setting.

>you will add Forcr CPU-Blit

>who use these buffers options

>Firmware Settings and Firmware Libraries should also be on Advanced IMO

I talked about what to move and not to move with various developers before submitting the pull request, and the selection I have in this PR is the conclusion of those talks. Once this PR is in master you can propose moving of additional settings in following pull requests.

**Pull request merged**